rules/Packet_UnwrapGREHTTP.lua

```diff
@@ -0,0 +1,46 @@
+function UnwrapGREHTTP (dpiMsg, packet)
+ -- rule content
+    require 'LOG'
+    require 'HexString'
+    
+    if packet ~= nil and GetLatestApplication(dpiMsg) == "http" then

```

Comment:
I'm not familiar with GRE. Is it always wrapped inside `http`? 
In case there might be another application on top of `http` then you might want to consider to use instead `bool HasApplication(dpiMessage, application)`

It is marked as a `flow` and not a `packet` API. This is wrong and I'll open up a bug with the team.  The `HasApplication` API call can be used on both flow and packet rules.

Reply:
GRE can encapsulate any layer 3 traffic and any application that works over a network (in our case it is only ever HTTP or HTTPS) but it could be different for other users. This may be useful. Is there any way to have an accepted list of protocols like below (without creating a complex if statement):

if HasApplication(dpiMsg, {'http','https'})

Sometimes, traffic gets classed as an application like Netflix of Sky_Player. This is web traffic, but has been classed as something more specific. Would this traffic be included with HasApplication(dpiMsg, 'https')?